src/main/java/jenkins/branch/MultiBranchProjectDisplayNamingStrategy.java

```diff
@@ -60,7 +60,14 @@ public String generateName(@NonNull final String rawName, final String displayNa
                 return rawName;
             }
 
-            return format("%s - %s", rawName, displayName);
+            // The raw name provided here in the context of pull requests is the pull request ID
+            // We tidy up the ID so that they display consistently between SCMs
+            String cleanedUpBranchName = rawName;
+            if (cleanedUpBranchName.startsWith("MR-") || cleanedUpBranchName.startsWith("PR-")) {
```

Comment:
may need a regex that it's PR-\d+ so its less likely to hit an actual branch name

Reply:
a branch name won't be hit here because the `rawName` and `displayName` would be the same so the above isn't a concern.